scripts.js

```diff
@@ -257,21 +261,19 @@ function pufflesGetItemById(label, itemTypeId) {
 }
 
 function pufflesSearchById(itemId) {
-  return pufflesItemsData.filter(function (pufflesItemsData) {
+  return pufflesItemsData.find(function (pufflesItemsData) {
```

Comment:
Thank you so much for this! :) The only issue with this change is although you also updated the first instance, there's a second here, which I think needs to become `pufflesSearchById(itemId).label` instead:

https://github.com/Aurorum/club-penguin-wordpress-plugin/blob/d9f2984609a84c9f936cfd559c82a192c8e3bbee/scripts.js#L199

That's just currently breaking it for now when there's an item ID stored and someone re-visits the page.

<img width="1318" alt="Screenshot 2020-06-20 at 12 51 15" src="https://user-images.githubusercontent.com/43215253/85201113-be23c100-b2f4-11ea-9d04-261ef7dfd3c7.png">